### PR TITLE
Updated et al numbers to match specs

### PR DESCRIPTION
**Pull request by rmhowie**
Specs: http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1945-5100/homepage/ForAuthors.html

### CODE REVIEW

**Comment by rmhowie** [comment]
This change also modifies the bibliography, but I only wanted to change the et al. settings for in text citations. 

This is probably a Zotero issue, maybe they don't support different styles for the in text and bibliography

File: meteoritics-and-planetary-science.csl [hunk 1]
```diff
@@ -112,7 +112,7 @@
     <text term="in" text-case="capitalize-first" suffix=" "/>
     <text variable="container-title" font-style="italic"/>
   </macro>
-  <citation et-al-min="10" et-al-use-first="1" et-al-subsequent-min="6" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+  <citation et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
```
Comment (adam3smith) [inline meteoritics-and-planetary-science.csl]: Please do:
`<citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">`
I realize they say "two or more" in their instructions, but that's not what they mean (check the journal -- in text citation of two authors are (firstauthor and secondauthor year). The subsequent bit doesn't do anything.

**Comment by adam3smith** [comment]
Works fine for me in Zotero.

**Comment by rmhowie** [comment]
> I realize they say "two or more" in their instructions, but that's not what they mean (check the journal -- in text citation of two authors are (firstauthor and secondauthor year).

Oh yeah, you're right!

**Comment by adam3smith** [comment]
great, thanks!

**Pull request merged**